Complain when someone tries to screw us with an invalid admin address.

--- mailman-2.0.9/Mailman/MailList.py	Tue May 29 10:45:27 2001
+++ mailman-2.0.9/Mailman/MailList.py	Mon Apr  8 18:28:18 2002
@@ -942,6 +942,8 @@
             self.web_page_url = mm_cfg.DEFAULT_URL
         if self.web_page_url and self.web_page_url[-1] <> '/':
 	    self.web_page_url = self.web_page_url + '/'
+        if '%s-admin@%s' % (self._internal_name, self.host_name) in self.owner:
+	    raise Errors.MMBadEmailError
 
     def IsListInitialized(self):
 	if not self._ready:
